src/main/java/com/kiwigrid/helm/maven/plugin/InitMojo.java

```diff
@@ -85,20 +99,22 @@ protected void downloadAndUnpackHelm() throws MojoExecutionException {
 			getLog().info("Found helm executable, skip init.");
 			return;
 		}
-
+		
```

Comment:
pls. re-format this file to get rid of orphaned blanks (there are some more, see line 106ff for example).

Once this is done, I'm glad to merge :)